api/services/reviewService.js

```diff
@@ -0,0 +1,52 @@
+const { reviewDao } = require("../models");
+const { commentValidate } = require("../utils/validate-check");
+const { reviewValidate } = require("../utils/validate-check");
+const { userMovieValidate } = require("../utils/validate-check");
+const jwt = require("jsonwebtoken");
+
+const createReview = async (accessToken, rating, comment, movieId) => {
+  if (!rating || !comment || !movieId) {
+    const error = new Error("KEY_ERROR");
+    error.statusCode = 400;
+    throw error;
+  }
+
+  if (rating < 1 || rating > 5) {
+    const error = new Error("INVALID_RATING");
+    error.statusCode = 400;
+    throw error;
+  }
+
+  commentValidate(comment);
+
+  const userId = jwt.verify(accessToken, process.env.JWT_SECRET).id;
+  const checkUserMovie = await reviewDao.checkUserMovie(userId, movieId);
+  await userMovieValidate(checkUserMovie);
+  return await reviewDao.createReview(userId, rating, comment, movieId);
+};
+
+const getReview = async (movieId) => {
+  return await reviewDao.getReview(movieId);
+};
+
+const updateReview = async (comment, movieId, accessToken) => {
+  commentValidate(comment);
+  const userId = jwt.verify(accessToken, process.env.JWT_SECRET).id;
+  const checkReview = (await reviewDao.checkReview(userId, movieId))[0].id;
+  await reviewValidate(checkReview);
```

Comment:
```suggestion
  if(!checkReview) throw new Error(. . .)
```
validate-check.js에서 드린 것과 동일한 내용입니다.